README.md

```diff
@@ -353,8 +353,8 @@ There are two special functions called periodically by the event loop:
 Inside server.c you can find code that handles other vital things of the Redis server:
 
 * `call()` is used in order to call a given command in the context of a given client.
-* `activeExpireCycle()` handles eviction of keys with a time to live set via the `EXPIRE` command.
-* `freeMemoryIfNeeded()` is called when a new write command should be performed but Redis is out of memory according to the `maxmemory` directive.
+* `activeExpireCycle()` handles eviciton of keys with a time to live set via the `EXPIRE` command.
```

Comment:
```suggestion
* `activeExpireCycle()` handles eviction of keys with a time to live set via the `EXPIRE` command.
```
i suppose this was an unintended change.

Reply:
I had gotten a merge conflict here.  I was intending to fix the spelling error.  Will fix.